index.html

```diff
@@ -5007,13 +4983,259 @@ <h3 id="looping-AudioBufferSourceNode">
+                                </li>
+                              </ol>
+                            </li>
+                          </ol>
+                        </li>
+                        <li>Emit the result of
+                        <code>playbackSignal(effectiveBufferTime)</code> as the
+                        output frame at <code>index</code>.
+                        </li>
+                        <li>Increase <code>bufferTime</code> by <code>dt *
+                        computedPlaybackRate</code>.
+                        </li>
+                      </ol>
+                    </li>
+                    <li>Increase <code>index</code> by 1.
+                    </li>
+                    <li>Increase <code>currentTime</code> by <code>dt</code>.
+                    </li>
+                  </ol>
+                </li>
+                <li>If <code>currentTime</code> &gt;= <code>stop</code>,
+                consider playback for the node to have ended.
+                </li>
+              </ol>
+            </li>
+          </ol>
+          <p>
+            The following figures illustrate the behavior of the algorithm in
+            assorted key scenarios. Buffer optimization is not considered, but
+            by definition it does not materially affect the resulting playback.
+            In all figures, the following conventions apply:
+          </p>
+          <ul>
+            <li>context sample rate is 1000 Hz
+            </li>
+            <li>buffer samples are shown with the first sample frame at the
+            <em>x</em> origin.
+            </li>
+            <li>output samples are shown with the <code>start</code> context
+            time sample rate interpolation.
+            </li>
+          </ul>
+          <p>
+            Basic playback with a simple loop that ends after the last sample
+            frame in the buffer:
+          </p>
+          <figure>
+            <img alt="AudioBufferSourceNode basic playback" src=
+            "images/absn-basic.png" width="556" height="485">
+            <figcaption>
+              <a>AudioBufferSourceNode</a> basic playback
+            </figcaption>
+          </figure>
+          <p>
+            <code>playbackRate</code> interpolation, illustrating half-speed
+            playback of buffer contents in which every other output sample
+            frame is interpolated. Of particular note is the last sample frame
+            in the looped output, which is interpolated using the loop start
+            point:
+          </p>
+          <figure>
+            <img alt="AudioBufferSourceNode playbackRate interpolation" src=
+            "images/absn-slow-loop.png" width="543" height="472">
+            <figcaption>
+              <a>AudioBufferSourceNode</a> playbackRate interpolation
+            </figcaption>
+          </figure>
+          <p>
+            Sample rate interpolation, illustrating playback of a buffer whose
```

Comment:
I'm not sure this diagram and text is needed.  The algorithm is specified in terms of time in the buffer instead of samples, so it's pretty clear that some kind of interpolation is needed if the rates don't match.